synctl

```diff
@@ -298,11 +317,16 @@ def main():
     action = options.action
 
     if action == "stop" or action == "restart":
+        hasStopped = False
         for worker in workers:
-            stop(worker.pidfile, worker.app)
+            if stop(worker.pidfile, worker.app):
+                hasStopped = True
```

Comment:
I don't think this will work properly for multiple workers, I suspect we want `has_stopped` to be `True` only if all workers stop successfully?

Reply:
I agree. I've updated the logic